bin/preinstall

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+if hash lerna 2>/dev/null; then
+  lerna bootstrap
+else
+  echo "Installing lerna. https://github.com/devtools-html/debugger.html/blob/master/docs/lerna.md"
+  npm i -g jasonlaster/lerna
```

Comment:
I don't think we should automatically install something globally on someone's machine. I think it'd be better to display a message telling the user to do `npm install -g jasonlaster/lerna`, particularly since this is a fork. I'm pretty they already have it installed this will overwrite the previous one.